Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-anomalydetector/README.md

### How to use

#### nodejs - Authentication, client creation and entireDetect as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample code lets you to determine anamolies with the given time series. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/anomaly-detector/)

Reviewer: ```suggestion
The following sample determines anamolies with the given time series. To know more, refer to the [Azure Documentation on Anomaly Detectors](https://docs.microsoft.com/en-us/azure/cognitive-services/anomaly-detector/)
```
